aea/protocols/dialogue/base.py

```diff
@@ -28,15 +28,25 @@
 import itertools
 import secrets
 from abc import ABC
+from collections import namedtuple
 from enum import Enum
 from inspect import signature
 from typing import Callable, Dict, FrozenSet, List, Optional, Set, Tuple, Type, cast
 
 from aea.common import Address
-from aea.exceptions import enforce
+from aea.exceptions import AEAEnforceError, enforce
 from aea.protocols.base import Message
 
 
+DialogueMessage = namedtuple(
+    "DialogueMessage",
+    ["performative", "contents", "is_incoming", "target"],
+    rename=False,
+    defaults=[None, None],
```

Comment:
should default of `is_incoming` not be a boolean? (edit: I see this is not possible based on usage below)

Reply:
what about default for contents being `{}`/`dict`?

Reply:
Done